### ISSUE TITLE
Suggestion: aggro toggling as unit saving mechanism

### ISSUE THREAD

**Issue by Slayer95** [SMUnlimited/AMAI#377]
- When Neutral Hostiles attack a unit controlled by AMAI, causing its HP to drop below a critical HP level, order it to attack a healthy ally unit nearby (without Healing Salve active) once.
- When (such) a healthy AI unit is attacked by their ally, order the attacker to ``Stop``.

This will leverage neutral hostiles built-in AI, causing them to stop attacking the unit in danger.

**Comment by jzy-chitong56** [comment]
In fact, the biggest difference between scripts and triggers is the inability to monitor in real-time, which makes it difficult to trigger intervention in a script in a timely manner. In theory, rescuing code to JOB care of low health units is already sufficient

**Comment by Slayer95** [comment]
Yeah. I worded my post in trigger-style. The commander could do such steps, but I don't think that would be the best solution. However, I am tempted to incorporate this routine in my own maps.
